### ISSUE TITLE
Views module support

### ISSUE THREAD

**Issue by vacho** [christophegalli/entity_d8#3]
Created entities can not be read by the views module

**Comment by christophegalli** [comment]
Yes, that is correct. Views support has not been implemented in this demo module. It is not in the scope of this documentation yet.

You can add that easily by implementing https://api.drupal.org/api/views/views.api.php/function/hook_views_data/7.

**Comment by christophegalli** [comment]
Hello
Thanks for adding to the documentation.

If you are interested, you can pull this https://github.com/christophegalli/learning_entity8 module which I did for the learning. It has the hook_views_data implemented. According to my buddy, this will automatically be included in core. Eventually :-)

**Comment by vacho** [comment]
Thanks @christophegalli this is a fantastic code.
